app/helpers/users_helper.rb

```diff
@@ -1,2 +1,20 @@
 module UsersHelper
+  def render_user_posts(user)
+    posts_html = ''
+    user.posts.each do |post|
+      posts_html += content_tag(:div, class: 'post') do
+        post_title = link_to post.Title, user_posts_path(user_id: user.id), class: 'custom-link-s'
+        post_content = content_tag(:p, post.Text, class: 'post-text')
+        text_div = content_tag(:div, class: 'text') do
+          comment_counter = content_tag(:p, "Comments: #{post.CommentsCounter.nil? ? 0 : post.CommentsCounter}",
+                                        class: 'comment-text')
+          likes_counter = content_tag(:p, "Likes: #{post.LikesCounter.nil? ? 0 : post.LikesCounter}",
+                                      class: 'likes-text')
+          comment_counter + likes_counter
+        end
+        post_title + post_content + text_div
+      end
```

Comment:
- [ ] [Optional] [Highly recommended] You can use partials for repeatable parts of your views, instead of what you used here `helpers` see [this](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) for more info.